src/locale/deviceLocales.ts

```diff
@@ -41,17 +41,23 @@ export function getLocales() {
     }
 
     if (typeof locale.languageTag === 'string') {
-      if (locale.languageTag.startsWith('zh-Hans')) {
-        // Simplified Chinese to zh-CN
-        locale.languageTag = 'zh-CN'
+      if (
+        locale.languageTag === 'zh-CN' ||
+        locale.languageTag.startsWith('zh-Hans')
```

Comment:
We still need to do this, because iOS will not automatically fall back from `zh-Hans-CN` to `zh-Hans`, using `zh-Hans-CN` would break browser compatibility.